inc/saiacl.h

```diff
@@ -2206,7 +2206,7 @@ typedef enum _sai_acl_entry_attr_t
     SAI_ACL_ENTRY_ATTR_ACTION_START = 0x00002000,
 
     /**
-     * @brief Redirect Packet to a destination which can be a port,
+     * @brief Redirect Packet to a destination which can be a port, system port,
```

Comment:
what about attribute SAI_PORT_ATTR_INGRESS_ACL/SAI_PORT_ATTR_EGRESS_ACL/... . Are they applicable for VOQ system ? In VOQ system where ACL will be binded SAI_PORT or SAI_SYSTEM_PORT

Reply:
Thought was to add capability where it didn't exist today. PORT object can be used to INGRESS and EGRESS qualifiers. To be able to redirect or monitor port on a different line card, system port needed to be added.